ocs_ci/ocs/utils.py

```diff
@@ -1630,6 +1630,55 @@ def get_primary_cluster_config():
             return cluster
 
 
+def get_recovery_cluster_config():
+    """
+    Get the recovery cluster config object in a DR scenario
+
+    Return:
+        framework.config: primary cluster config obhect from config.clusters
+
+    """
+    for cluster in ocsci_config.clusters:
+        if cluster.MULTICLUSTER.get("recovery_cluster"):
+            return cluster
+
+
+def set_recovery_as_primary():
```

Comment:
While testing this PR, please pay attention to how the follow up tests and teardown of the cluster behaves as we have modifed cluster entries.

Reply:
Taken care after reindexing.